src/Exthost/Eol.re

```diff
@@ -0,0 +1,12 @@
+[@deriving (show({with_path: false}), yojson({strict: false}))]
```

Comment:
This - along with the `ModelContentChange`/`ModelChangeEvent` modules were ported from the existing protocol. In the interest of saving time, so I can get us wired up to the v2, I'm going to keep as-is and not convert them to the decoder approach.